src/helpers/git_helpers/branch_helpers.sh

```diff
@@ -146,6 +182,15 @@ function delete_local_branch_needs_force {
 }
 
 
+# Returns the previously checked out branch name
+function previous_branch_name {
+  git rev-parse --abbrev-ref "@{-1}" > /dev/null 2>&1
+  if (($? == 0)); then
+    git rev-parse --abbrev-ref "@{-1}"
```

Comment:
What does running the exact same command give us here that it didn't give us running the first time on line 187?